packages/foreman/rubygem-sassc/rubygem-sassc.spec

```diff
@@ -1,12 +1,14 @@
 # template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
+%{!?_root_libdir:%global _root_libdir %{_libdir}}
```

Comment:
So this ends up as `/usr/lib`, not as `/usr/lib64` and it explodes from there.

Reply:
https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/ suggests that `%{_libdir}` should be `/usr/lib64` on 64 bit machines. Is that different on EL8?

Reply:
No, `rpm -E '%{_libdir}` does lib64 on my F31, EL7 and EL8. But Koji for some reason ends up with lib. :woman_shrugging:

Reply:
on both, el7 and el8 build that is!

Reply:
@pcreech any idea?

Reply:
And to make matters worse, it is lib64 when I run it in mock with our el7-scl config.

Reply:
That explains why it worked when I debugged it locally using mock.

Reply:
maybe because it's a noarch build? :)

Reply:
yepp, friggin noarch